src/qldb_helper/qldb_helper/__init__.py

```diff
@@ -0,0 +1,139 @@
+from typing import Protocol
+
+from pyqldb.cursor.stream_cursor import StreamCursor
+
+
+class QLDBTransactionExecutor(Protocol):
+    def execute_statement(self, statement: str, *parameters) -> StreamCursor:
+        pass
+
+
+class Driver(Protocol):
+    def execute_lambda(self, lambda_func: ()) -> any:
+        pass
+
+
+class QLDBError(Exception):
+    pass
+
+
+class InsertTransactionError(QLDBError):
+    def __init__(self, exception: Exception):
+        super().__init__(f'Error inserting into transactions exception:{exception}')
+
+
+class UpdateBalanceError(QLDBError):
+    def __init__(self, exception: Exception):
+        super().__init__(f'Error updating into balances - exception:{exception}')
+
+
+class GetBalanceError(QLDBError):
+    pass
+
+
+class QLDBHelper:
+    @classmethod
+    def get_balance(cls, sub: str, executor: QLDBTransactionExecutor) -> int:
+        try:
+            cursor = executor.execute_statement("SELECT balance FROM balances WHERE sub = ?", sub)
+            first_record = next(cursor, None)
+            if first_record:
+                return first_record["balance"]
+            else:
+                raise GetBalanceError(f"Unable to get balance for user:{sub}")
+        except Exception as e:
+            raise e
+
+    @classmethod
+    def insert_transaction(cls, values: dict, executor: QLDBTransactionExecutor):
+        """
+        :type values: dict
+        :param values: a dictionary of the values to insert
+
+        :type executor: lambda executor
+        :param executor: object that contains the function execute_statement
+
+        :raises InsertError: If insert fails
+
+        :return: id of the inserted record
+
+        """
+        transaction_id = values.get("id")
```

Comment:
Similar comment here, because you know that the structure should be a certain way it's probably worth building a type / class here